README.md

```diff
@@ -102,9 +102,6 @@ If you want to upload to the aws s3 bucket, you can no longer use your local key
 - AUTH_SECRET=\<insert the secret for the tdr client in the integration account\>
 - IDENTITY_POOL_ID=secret-from-/mgmt/identitypoolid_intg in the management account
 
-You will also need to set the AUTH_URL environment variable for your locally running API project to:
--  AUTH_URL=https://auth.tdr-integration.nationalarchives.gov.uk/auth
```

Comment:
This step wasn't necessary in your case because you were running a fully local development environment, but it is necessary if you follow the step just before it, and connect to a local API and the integration auth server.

This setup will get simpler once I've finished setting up some purely local backend checks, so it might be easiest to leave this for now.

Reply:
Ah whoops! I'll roll that back, should I add a note to say when it should be used?

Reply:
I think it's fine to close this PR. The current docs are a bit confusing, but I'll be making some sweeping changes to them soon when I've finished the local dev environment setup.